examples/nextjs/src/components/content-types/blog.js

```diff
@@ -1,12 +1,12 @@
-import { isInsideEditor } from "@dotcms/client";
-import { BlockEditorRenderer } from "@dotcms/react";
+import { isEditing } from '@/utils/isEditing';
```

Comment:
Why? did we change the config to single quotes?

Reply:
My IDE is set to single quotes, we do not have a prettier file nor `editor.config`

Reply:
We need to do something, it keeps happening.

Reply:
Not sure which one can be less intrusive. `editor.config` or `prettier`